presets/massa-station-preset.js

```diff
@@ -39,6 +39,14 @@ module.exports = {
         neutral: colorDarkBlue,
         info: colorLightBlue,
         success: colorBrand,
+        default_primary: colorDarkBlue,
+        hover_primary: colorBasicBlue,
+        pressed_primary: colorLightBlue,
+        disabled_primary: colorDarkGrey,
+        default_secondary: colorWhite,
+        hover_secondary: colorDarkBlue,
+        pressed_secondary: colorDarkBlue,
+        disabled_secondary: colorLightGrey,
```

Comment:
Why do we need that!? Colors are already there!? 🤔

Reply:
![image](https://user-images.githubusercontent.com/72019005/235875793-e43fa8bd-4169-4129-8b3c-832813920351.png)

Lucas added all states in theme.
As the colours between primary and secondary are not the same for the states, I put them in the theme.
Do you have any suggestion to improve this?

Reply:
![image](https://user-images.githubusercontent.com/72019005/235877606-3caf5d4a-fd93-4951-b292-5c4a75abf9de.png)

I suggest, we can add it in addComponents section (look on screenshot) like this :
`
".btn-primary-default": {
          background: colorWhite,
        },
`

Same for all states.

Reply:
Ah got it!

I would suggest to ask Lucas about the color names and add that on the colors presset. Please, keep @Thykof aware of that too. (avoid _ on names too - please camelCase here in JS).

Reply:
Okay!
So if I understand correctly, we're sticking with what I did. But I change the "default_primary" to "btnPrimaryDefault"?

Or

I add ".btn-primary-default" (for example) in addComponent function to use it in the button component ?

Reply:
No, not exactlu btnPrimaryDefault. We can't have colors names that rely on components or action names. It needs to be more generic. Please, sync with Paul about that. I just explain that to him.

Reply:
Ok ! I will ask Paul to explain.

Thank you for your time ;)